pycsw/core/util.py

```diff
@@ -33,11 +33,15 @@
 import time
 import datetime
 import logging
-import urllib2
+try:
+    from urllib2 import Request, urlopen
+except:
+    from urllib.request import Request, urlopen
 from shapely.wkt import loads
 from owslib.util import http_post
 from pycsw.core.etree import etree
 
+from six import text_type
```

Comment:
> pycsw doesn't import six, and OWSLib does, can pycsw still use it?

Yes, but it would be better to be explicit and import it.  The alternative, to avoid adding an extra dependency, is to drop the `six.py` file there.  That **is** one of the recommended ways to use six.


> If pycsw doesn't use six, what are the implications?

It is just a little bit harder to  make code compliant with both python 2 and 3.  Six also makes the code more readable.